pyTigerGraph/gds/dataloaders.py

```diff
@@ -1261,6 +1264,20 @@ def _start(self) -> None:
 
         raise NotImplementedError
 
+    def stop(self, remove_topics=False) -> None:
```

Comment:
It would be safer to call the _reset function as that cleans all the threads as well. Pass `theend=True` to also delete the Kafka topics